esdb/operations.go

```diff
@@ -0,0 +1,16 @@
+package esdb
+
+type ServerVersion struct {
+	Major int
+	Minor int
+	Patch int
+}
+
+func (client *Client) GetServerVersion() (ServerVersion, error) {
```

Comment:
Sorry I saw it after but now that it's publicly exposed, could write some documentation for the `GetServerVersion` function and the `ServerVersion` struct, please?

Reply:
will do :+1:

Reply:
fixed.